src/amo/components/SearchForm.js

```diff
@@ -69,4 +74,7 @@ export function mapDispatchToProps(dispatch) {
   };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(SearchFormBase);
+export default compose(
+  connect(mapStateToProps, mapDispatchToProps),
+  translate({ withRef: true }),
```

Comment:
I know we seem to use `withRef: true` all over but I'm not sure we actually need it.